### DISCUSSION TITLE
RobiniaDocs API Explorer

### DISCUSSION THREAD

**Discussion by NeuroXiq** [saucecontrol/PhotoSauce#122] [Ideas]
Maybe You will be interested in hosting API Explorer online: 
[www.robiniadocs.com/d/photosauce](https://www.robiniadocs.com/d/photosauce/api/PhotoSauce.MagicScaler.MagicImageProcessor.html?q=processimage)

**Comment by saucecontrol** [comment]
I'm already hosting DocFX-generated docs:
https://docs.photosauce.net/api/PhotoSauce.MagicScaler.MagicImageProcessor.html

The formatting is nice on your link, minus the lack of support for all of the DocFX tagging, e.g.

![image](https://github.com/saucecontrol/PhotoSauce/assets/7929102/57a631d4-aaad-4b05-ba9a-6a8780ce3907)

What are you suggesting I do?

**Reply by NeuroXiq** [reply]
1. Yes i fixed this issue thanks

Sorry I didn't noticed that You are already hosting Your own Docfx documentation, so seems that this does not make sens to host again on RobiniaDocs
Maybe You will find Robinia usefull for some other projects :)
I think that we can close this discussion, thanks for response and have a nice day :)
